examples/apps/face_detection/main.py

```diff
@@ -28,11 +28,11 @@
         movie_name + '_bboxes')
 
     print('Drawing faces onto video...')
-    frame = db.ops.FrameInput()
+    frame = db.sources.FrameColumn()
     sampled_frame = frame.sample()
-    bboxes = db.ops.Input()
+    bboxes = db.sources.Column()
     out_frame = db.ops.DrawBox(frame = sampled_frame, bboxes = bboxes)
-    output = db.ops.Output(columns=[out_frame])
+    output = db.sinks.Column(columns={'frame': out_frame})
```

Comment:
We should probably have a separate `sinks.Sequence()` and `sinks.Table([list of (name, node)])` where sinks.Sequence can be a table with a single anonymous column.

Reply:
Alternatively we could just get rid of the `sinks.Table` and only have `sinks.Sequence` since we don't really use the multi-column aspect of it.

Reply:
Fine with that too!

Reply:
The reason I didn't do that in this PR is because we don't currently have support for multiple outputs. I could collapse them into one stage like we do for inputs, but haven't implemented it yet.

As for the name, I liked `Column` since it suggests persistent storage, where as `Sequence` doesn't as much. Can you think of a better name?